sqlglot/dialects/spark2.py

```diff
@@ -177,31 +177,34 @@ class Generator(Hive.Generator):
         TRANSFORMS = {
```

Comment:
Added `exp.WithinGroup` transform, the rest were just sorted in alphabetical order.